src/movies.js

```diff
@@ -1,16 +1,69 @@
+
+
 // Iteration 1: All directors? - Get the array of all directors.
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors.
 // How could you "clean" a bit this array and make it unified (without duplicates)?
-function getAllDirectors() {}
+
+
+function getAllDirectors(moviesArr) {
+  const totalDirectors = movies.map(function(filmObj){
```

Comment:
You pass 'moviesArr' as argument, this array contains your movies data that you pass into your function -> OK
Then you use map method on 'movies' variable -> what does this array contain? ;)

Be consistent with the names you give.  If function takes moviesArr as parameter, that is what you will be using.  eg. moviesArr.map()

Review same issue in all the iterations ;)